lib/hanami/utils/string.rb

```diff
@@ -151,6 +152,10 @@ def classify
         self.class.new words.zip(delimiters).join
       end
 
+      def classified?
+        split(NAMESPACE_SEPARATOR).reduce(false) {|memo, str| str.match(/\A[A-Z][a-zA-Z_]*/).to_s.length == str&.length }
```

Comment:
Also, classes can legally have numbers on it's names, so `\w` would be more suitable as @davydovanton said. =)